src/build.py

```diff
@@ -276,1424 +284,1499 @@ def JavaBin():
-             checkout=GCC_REVISION, depth=GCC_CLONE_DEPTH),
-      Source('v8', work_dirs.GetV8(),
-             GIT_MIRROR_BASE + 'v8/v8.git',
-             custom_sync=ChromiumFetchSync),
-      Source('host-toolchain', work_dirs.GetV8(), '',
-             custom_sync=SyncToolchain),
-      Source('cmake', '', '',  # The source and git args are ignored.
-             custom_sync=SyncPrebuiltCMake),
-      Source('nodejs', '', '',  # The source and git args are ignored.
-             custom_sync=SyncPrebuiltNodeJS),
-      Source('gnuwin32', '', '',  # The source and git args are ignored.
-             custom_sync=SyncGNUWin32),
-      Source('wabt', GetSrcDir('wabt'),
-             WASM_GIT_BASE + 'wabt.git'),
-      Source('binaryen', GetSrcDir('binaryen'),
-             WASM_GIT_BASE + 'binaryen.git'),
-      Source('wasi-libc', GetSrcDir('wasi-libc'),
-             'https://github.com/CraneStation/wasi-libc.git'),
-      Source('java', '', '',  # The source and git args are ignored.
-             custom_sync=SyncPrebuiltJava),
-      Source('sysroot', '', '',  # The source and git args are ignored.
-             custom_sync=SyncLinuxSysroot)
-  ]
+    return [
+        Source('waterfall', SCRIPT_DIR, None, custom_sync=NoSync),
+        Source('llvm', GetSrcDir('llvm-project'),
+               LLVM_GIT_BASE + 'llvm-project.git'),
+        Source('llvm-test-suite', GetSrcDir('llvm-test-suite'),
+               LLVM_MIRROR_BASE + 'llvm-test-suite.git'),
+        Source('emscripten', GetSrcDir('emscripten'),
+               EMSCRIPTEN_GIT_BASE + 'emscripten.git'),
+        Source('fastcomp', GetSrcDir('emscripten-fastcomp'),
+               EMSCRIPTEN_GIT_BASE + 'emscripten-fastcomp.git'),
+        Source('fastcomp-clang', GetSrcDir('emscripten-fastcomp-clang'),
+               EMSCRIPTEN_GIT_BASE + 'emscripten-fastcomp-clang.git'),
+        Source('gcc',
+               GetSrcDir('gcc'),
+               GIT_MIRROR_BASE + 'chromiumos/third_party/gcc.git',
+               checkout=GCC_REVISION,
+               depth=GCC_CLONE_DEPTH),
+        Source('v8',
+               work_dirs.GetV8(),
+               GIT_MIRROR_BASE + 'v8/v8.git',
+               custom_sync=ChromiumFetchSync),
+        Source('host-toolchain',
+               work_dirs.GetV8(),
+               '',
+               custom_sync=SyncToolchain),
+        Source('cmake',
+               '',
+               '',  # The source and git args are ignored.
+               custom_sync=SyncPrebuiltCMake),
+        Source('nodejs',
+               '',
+               '',  # The source and git args are ignored.
+               custom_sync=SyncPrebuiltNodeJS),
+        Source('gnuwin32',
+               '',
+               '',  # The source and git args are ignored.
+               custom_sync=SyncGNUWin32),
+        Source('wabt', GetSrcDir('wabt'), WASM_GIT_BASE + 'wabt.git'),
+        Source('binaryen', GetSrcDir('binaryen'),
```

Comment:
Why did it expand some of these and not all of them?

Reply:
I'dd take a look and see if I can come up with something better.  I don't know all of the yapf rules that cause this kind of thing.